frontend/src/components/NavBar.tsx

```diff
@@ -0,0 +1,32 @@
+import Typography from '@mui/material/Typography';
+import AppBar from '@mui/material/AppBar';
+import PersonIcon from '@mui/icons-material/Person';
+import KnowItLogo from './knowit.svg';
+import AVALogo from './ava.svg';
+
+const NavBar = () => {
+    return (
+        <AppBar
+            className="navigation"
+            elevation={0}
+            style={{ backgroundColor: '#333333', position: 'relative' }}
```

Comment:
This styling should be moved to the SCSS file.

Reply:
Also, you should `@import '../styles/_colors';` and use the color variables defined there. No need to define them in each file.